### ISSUE TITLE
namespace prism css to avoid collisions with component css

### ISSUE THREAD

**Issue by dusty** [doczjs/docz#1032]
I have a component that is using the bulma css framework.  In docz 2.0.0-rc.5 when I view the source of the component, the styles are overridden by bulma which also targets a .tag class name.

Example:
<img width="1069" alt="Screen Shot 2019-08-28 at 12 28 57 PM" src="https://user-images.githubusercontent.com/792/63899027-298c5300-c9b0-11e9-8083-baf13e46f402.png">

If I remove the CSS import, the view source looks great, but of course my component doesn't anymore.

<img width="967" alt="Screen Shot 2019-08-28 at 12 43 09 PM" src="https://user-images.githubusercontent.com/792/63899046-4032aa00-c9b0-11e9-94ac-c76096071062.png">

It would be nice if prism highlighter could be prefixed so its css styling could work independently of whatever CSS the components docz is showing uses.

**Comment by rakannimer** [comment]
I think this more of an issue than enhancement. Happy to help anyone who would like to take this on !

**Comment by rakannimer** [comment]
Hey @dusty   

This is solved in 2.2.0. 

To scope your playground styles and make them independent of docz theme css you have 2 choices : 

1. add a useScoping prop to your playground

```jsx
<Playground useScoping> 
{
// blablabla
}
</Playground>
```
2. add `useScopingInPlayground : true` in doczrc.js 

```js
export default {
	useScopingInPlayground: true
}
```

Sorry for the delay in the reply but the fix was implemented thanks to you raising this issue ! So thanks 👍 

Going to close it, please re-open or comment if for some reason something doesn't work for you !

If you're interested in implementation details check #1305
